<?xml version="1.0"?>
<launch>
    <!-- config file path -->
    <arg name="yawrate_mean_calculator_settings" default="$(find ccv_data_maker)/config/param/yawrate_mean_calculator.yaml"/>

    <!-- node launch -->
    <node pkg="ccv_data_maker" type="yawrate_mean_calculator" name="yawrate_mean_calculator" output="screen">
        <rosparam command="load" file="$(arg yawrate_mean_calculator_settings)"/>
    </node>
    <node pkg="rosbag" type="play" name="play" args="$(env HOME)/bag/ccv40.bag --clock"/>
</launch>